cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/util/AbstractAppFabricHttpHandler.java

```diff
@@ -620,16 +628,7 @@ private static boolean usesStream(FlowSpecification flowSpec, String stream) {
 
   /* -----------------  helpers to return Json consistently -------------- */
 
-  protected static final DatasetRecord makeDataSetRecord(String name, String classname) {
-    return new DatasetRecord("Dataset", name, name, classname);
-  }
-
-  protected static final StreamRecord makeStreamRecord(String name, StreamSpecification specification) {
-    return new StreamRecord("Stream", name, name, GSON.toJson(specification));
+  protected static StreamRecord makeStreamRecord(String name, StreamSpecification specification) {
```

Comment:
actually, StreamSpecification is just a name... so the api returns name 3 times? seems wrong though its existing behavior